doc/related_projects.rst

```diff
@@ -182,6 +182,9 @@ and tasks.
   (`CRFsuite <http://www.chokkan.org/software/crfsuite/>`_ wrapper with
   sklearn-like API).
 
+- `skforecast <https://github.com/JoaquinAmatRodrigo/skforecast>`_ A python library that eases using scikit-learn 
+  regressors as multi-step forecasters. It also works with any regressor compatible with the scikit-learn API.
```

Comment:
I know this file is not consistent, but I prefer trying to keep the character count to be less than <88 per line:

```suggestion
- `skforecast <https://github.com/JoaquinAmatRodrigo/skforecast>`_ A python library
  that eases using scikit-learn regressors as multi-step forecasters. It also works
  with any regressor compatible with the scikit-learn API.
```